ee-9/feature-pack/pom.xml

```diff
@@ -508,6 +508,8 @@
                                 <exclude>io.opentelemetry:opentelemetry-sdk-trace\z</exclude>
                                 <exclude>io.opentelemetry:opentelemetry-semconv\z</exclude>
                                 <exclude>io.perfmark:perfmark-api\z</exclude>
+                                <exclude>io.smallrye:smallrye-open-api-core</exclude>
+                                <exclude>io.smallrye:smallrye-open-api-jaxrs</exclude>
```

Comment:
@pferraro Is this needed because of the javax.annotation thing?

I asked in zulip too, as this is good to go if the answer is yes. If it's no then these should move out of this chunk and have a comment explaining why they are excluded.

Reply:
Paul's answer was yes. :)